src/osd/osd_types.cc

```diff
@@ -1483,7 +1483,8 @@ void pg_pool_t::encode(bufferlist& bl, uint64_t features) const
   }
 
   uint8_t v = 24;
-  if (!(features & CEPH_FEATURE_NEW_OSDOP_ENCODING)) {
+  if (!(features & CEPH_FEATURE_NEW_OSDOP_ENCODING) ||
+      !(features & CEPH_FEATURE_SERVER_JEWEL)) {
```

Comment:
but v0.94.9 does not have `CEPH_FEATURE_NEW_OSDOP_ENCODING` defined. so 
`!(features & CEPH_FEATURE_NEW_OSDOP_ENCODING)` is already true, when the monitor is talking to a v0.94.9 osd. am i right?